File: UsersModule/src/main/java/dawid/kotarba/users/UsersModule.java
package dawid.kotarba.users;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

@SpringBootApplication
@EnableEurekaClient
public class UsersModule {
    public static void main(String[] args) {
        SpringApplication.run(UsersModule.class, args);
    }
}

File: UsersModule/src/main/scala/dawid/kotarba/users/UsersModule.scala
package dawid.kotarba.users

import dawid.kotarba.users.config.MainConfig
import org.springframework.boot.SpringApplication

/**
  * Created by Dawid on 01.07.2016.
  */

object UsersModule extends App {
  SpringApplication.run(classOf[MainConfig])
}
